### PR TITLE
Feature/admin writeback olids to ia

### PR DESCRIPTION
**Pull request by mekarpeles**

### CODE REVIEW

File: openlibrary/catalog/add_book/__init__.py [hunk 1]
```diff
@@ -395,6 +405,56 @@ def add_cover(cover_url, ekey):
...
+
+def update_ia_metadata_for_ol_edition(edition_id):
+    """An ol_edition is of the form OL...M"""
+
+    data = {'error': 'No qualifying edition'}
+    if edition_id:
+        ed = web.ctx.site.get('/books/%s' % edition_id)
+        if ed.ocaid:
+            work = ed.works[0] if ed.get('works') else None
+            if work.key:
```
Comment (bfalling) [inline openlibrary/catalog/add_book/__init__.py]: Seems like `work.key` could throw an error if the previous line leads `work` to be set to `None`.

File: openlibrary/catalog/add_book/__init__.py [hunk 2]
```diff
@@ -568,4 +628,7 @@ def load(rec):
             assert isinstance(i, dict)
 
         web.ctx.site.save_many(edits, 'import new book')
+
+    # update_ia_metadata_for_ol_edition(reply['edition']['key'].split('/')[2])
```
Comment (bfalling) [inline openlibrary/catalog/add_book/__init__.py]: Should this comment stay?

File: openlibrary/plugins/admin/code.py [hunk 3]
```diff
@@ -202,7 +238,9 @@ def POST_block_account_and_revert(self, account):
         account.block()
         changes = account.get_recentchanges(limit=1000)
         changeset_ids = [c.id for c in changes]
-        ipaddress_view().revert(changeset_ids, "Reverted Spam")
+        ipaddress_view().revert(changeset_ids
+
```
Comment (bfalling) [inline openlibrary/plugins/admin/code.py]: Remove blank line?

File: openlibrary/catalog/add_book/__init__.py (hunk 2 above)
Comment (mekarpeles) [reply openlibrary/catalog/add_book/__init__.py]: placeholder next stage where we have importbot and addwork write back to IA automatically

**Pull request merged**